src/flamenco/runtime/fd_executor.c

```diff
@@ -47,6 +47,10 @@
 #define MAX_COMPUTE_UNITS_PER_BLOCK                (48000000UL)
 #define MAX_COMPUTE_UNITS_PER_WRITE_LOCKED_ACCOUNT (12000000UL)
 
+int fd_noop_instr_execute( fd_exec_instr_ctx_t * ctx FD_PARAM_UNUSED ) {
```

Comment:
nits:
should be int \n fd_noop_instr_execute and ( fd_exec_instr_ctx_t * FD_PARAM_UNUSED ctx )

Reply:
maybe also leave a todo saying that we need to move this back into fd_execute_instr